tests/cli/test_hooks.py

```diff
@@ -48,17 +62,34 @@ def test_composite_runs_all_hooks(project, invoke, capfd):
         "pre_first": {"shell": "echo 'Pre-First CALLED'"},
         "second": {"shell": "echo 'Second CALLED'"},
         "post_second": {"shell": "echo 'Post-Second CALLED'"},
+        "pre_script": {"shell": "echo 'Pre-Script CALLED'"},
+        "post_script": {"shell": "echo 'Post-Script CALLED'"},
+        "pre_run": {"shell": "echo 'Pre-Run CALLED'"},
+        "post_run": {"shell": "echo 'Post-Run CALLED'"},
```

Comment:
We may remove the single quotes to fix the CI failure on Windows.

Reply:
I reused the `_vars` fixture which is now `_echo`
Note to myself: I definitively need a Windows test platform !